docs/content/preview/migrate/yb-voyager/yb-voyager-cli.md

```diff
@@ -0,0 +1,311 @@
+        --source-db-type sourceDB \
+        --source-db-host localhost \
+        --source-db-user username \
+        --source-db-password password \
+        --source-db-name dbname \
+        --source-db-schema schemaName #Include schema name only for Oracle migrations.
+
+```
+
+### analyze-schema
+
+Analyse the PostgreSQL schema dumped in the export schema step.
+
+#### Syntax
+
+```sh
+yb_voyager analyze-schema [ <arguments> ... ]
+```
+
+- *arguments*: See [Arguments](#arguments)
+
+#### Example
+
+```sh
+yb-voyager analyze-schema --export-dir /path/to/yb/export/dir \
+        --source-db-type sourceDB \
+        --source-db-host localhost \
+        --source-db-user username \
+        --source-db-password password \
+        --source-db-name dbname \
+        --source-db-schema schemaName \ #Include schema name only for Oracle migrations.
+        --output-format txt
+```
+
+### export data
+
+Dump the source database to the machine where yb-voyager is installed.
+
+#### Syntax
+
+```sh
+yb_voyager export data [ <arguments> ... ]
+```
+
+- *arguments*: See [Arguments](#arguments)
+
+#### Example
+
+```sh
+yb-voyager export data --export-dir /path/to/yb/export/dir \
+        --source-db-type sourceDB \
+        --source-db-host hostname \
+        --source-db-user username \
+        --source-db-password password \
+        --source-db-name dbname \
+        --source-db-schema schemaName #Include schema name only for Oracle migrations.
+```
+
+### import schema
+
+Import schema to the target YugabyteDB.
+
+#### Syntax
+
+```sh
+yb_voyager import schema [ <arguments> ... ]
+```
+
+- *arguments*: See [Arguments](#arguments)
+
+#### Example
+
+```sh
+yb-voyager import schema --export-dir /path/to/yb/export/dir \
+        --target-db-host hostname \
+        --target-db-user username \
+        --target-db-password password \
+        --target-db-name dbname
+```
+
+### import data
+
+Import the data objects to the target YugabyteDB.
+
+#### Syntax
+
+```sh
+yb_voyager import data [ <arguments> ... ]
+```
+
+- *arguments*: See [Arguments](#arguments)
+
+#### Example
+
+```sh
+yb-voyager import data --export-dir /path/to/yb/export/dir \
+        --target-db-host hostname \
+        --target-db-user username \
+        --target-db-password password \
+        --target-db-name dbname \
+        --parallel-jobs connectionCount \
+        --batch-size size
+```
+
+### import data file
+
+Load all your data files in csv format directly to the target YugabyteDB.
+
+#### Syntax
+
+```sh
+yb_voyager import data file [ <arguments> ... ]
+```
+
+- *arguments*: See [Arguments](#arguments)
+
+#### Example
+
+```sh
+yb-voyager import data file --export-dir /path/to/yb/export/dir \
+        --target-db-host hostname \
+        --target-db-port port \
+        --target-db-user username \
+        --target-db-password password \
+        --target-db-name dbname \
+        –-data-dir “/path/to/files/dir/” \
```

Comment:
change the double quotes used: `"/path/to/files/dir/"` instead of `“/path/to/files/dir/”`
current quotes are identified as a character literal on linux terminal
similarly of `--file-table-map`